### PR TITLE
Submit SKIPPED status to vacuole

### PR DESCRIPTION
**Pull request by rahuldesai1**

### CODE REVIEW

File: latch_cli/utils/workflow.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    try:
+        token = s[0]
+        node_name = s[1]
+        retry = s[2]
+    except IndexError:
+        raise RuntimeError(f"Invalid pod name: {pod_name}")
+
+    arr_index = None if len(s) <= 3 else s[3]
+    arr_retry = None if len(s) <= 4 else s[4]
```
Comment (AidanAbd) [inline latch_cli/utils/workflow.py:19]: there are perhaps more fucked up than you think -- check what nucleus-logs does.

**Review by AidanAbd — APPROVED** [review]

File: latch_cli/utils/workflow.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+import gql
+from latch_sdk_gql.execute import execute
+
+pod_name_regex = re.compile(
+    r"^(?P<token>[a-zA-Z0-9]+)-(?P<node_name>(n\d+-\d+-)*n\d+)-(?P<retry>\d)+(?:-(?P<arr_index>\d+))?(?:-(?P<arr_retry>\d+))?$"
+)
+
+
+def override_task_status(status: str) -> None:
```
Comment (maximsmol) [inline latch_cli/utils/workflow.py:11]: should probably be made private?

File: latch_cli/utils/workflow.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import re
+
+import gql
+from latch_sdk_gql.execute import execute
+
+pod_name_regex = re.compile(
+    r"^(?P<token>[a-zA-Z0-9]+)-(?P<node_name>(n\d+-\d+-)*n\d+)-(?P<retry>\d)+(?:-(?P<arr_index>\d+))?(?:-(?P<arr_retry>\d+))?$"
```
Comment (maximsmol) [inline latch_cli/utils/workflow.py:7]: would prefer an extended regex like

```
r"""
^(
    (?P<token>[a-zA-Z0-9]+)
		-
		(?P<node_name>
				(
						n
						[0-9]+
						-
						[0-9]+
						-
				)*
				n
				[0-9]+
		)
		-
		...
)
"""
```

**Review by maximsmol — CHANGES_REQUESTED** [review]

File: latch_cli/utils/workflow.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        )?
+    )$
+    """,
+    re.VERBOSE,
+)
+
+
+def _override_task_status(status: str) -> None:
+    with open("/etc/hostname", "r") as f:
+        pod_name = f.read().strip()
```
Comment (ayushkamat) [inline latch_cli/utils/workflow.py:40]:
```suggestion
    pod_name = Path("/etc/hostname").read_text().strip()
```

**Review by ayushkamat — CHANGES_REQUESTED** [review]
smol things

**Pull request merged**